mappings/briefs-digital-outcomes-and-specialists-2.json

```diff
@@ -27,8 +27,8 @@
         "_": "DO NOT UPDATE BY HAND",
         "version": "10.0.0",
         "generated_from_framework": "digital-outcomes-and-specialists-2",
-        "generated_by": "digitalmarketplace-frameworks/scripts/generate-search-config.py",
-        "generated_time": "2017-11-24T10:51:58.947073",
+        "generated_by": "/digitalmarketplace-frameworks/scripts/generate-search-config.py",
```

Comment:
Weird, why do we get the slash now?

Reply:
Oh, it had my name and stuff in the path so I removed. just left it with a leading slash.

Reply:
take out the slash and we'll have a zero diff on that line :)

Reply:
s'gone!